### PR TITLE
Add management command for fixing forms with "undefined" xmlns (round 2)

### PR DESCRIPTION
**Pull request by NoahCarnahan**
This is an extension to the work done in https://github.com/dimagi/commcare-hq/pull/11330. The previous management command can't easily be reused (without modification) because it assumes that it's fixing both xforms and apps, and matching xforms to forms that don't have an xmlns.

My intention is to run this command now, and follow up with a PR that introduces celery task which will run this command periodically (and parse the log file it produces to generate email-able metrics about the results)

related tickets:
http://manage.dimagi.com/default.asp?229913
http://manage.dimagi.com/default.asp?228594

@czue cc @emord

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 6 errors:

* corehq/apps/cleanup/management/commands/fix_xforms_with_undefined_xmlns.py, line 212 - E501 line too long (140 > 115 characters)
* corehq/apps/cleanup/management/commands/fix_xforms_with_undefined_xmlns.py, line 224 - F821 undefined name 'build'
* corehq/apps/cleanup/management/commands/fix_xforms_with_undefined_xmlns.py, line 217 - E302 expected 2 blank lines, found 1
* corehq/apps/cleanup/management/commands/fix_xforms_with_undefined_xmlns.py, line 170 - E302 expected 2 blank lines, found 1
* corehq/apps/cleanup/management/commands/fix_xforms_with_undefined_xmlns.py, line 32 - E303 too many blank lines (3)
* corehq/apps/cleanup/management/commands/fix_xforms_with_undefined_xmlns.py, line 221 - F841 local variable 'app' is assigned to but never used

**Comment by NoahCarnahan** [comment]
(hold off on merge, messed something up in a rebase)

File: corehq/apps/cleanup/management/commands/fix_xforms_with_undefined_xmlns.py [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+        include_docs=True,
+    )
+    return [get_correct_app_class(row['doc']).wrap(row['doc']) for row in saved_apps]
+
+
+class MultiplePreviouslyFixedForms(Exception):
+    def __init__(self, build_id, app_id):
+        msg = "Unable to determine matching form. Multiple forms in app {} " \
+              "were previously repaired. Build submitted against was {}".format(
+            app_id, build_id
```
Comment (dimagimon) [inline corehq/apps/cleanup/management/commands/fix_xforms_with_undefined_xmlns.py]: E122 continuation line missing indentation or outdented

File: corehq/apps/cleanup/management/commands/fix_xforms_with_undefined_xmlns.py [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+        )
```
Comment (dimagimon) [inline corehq/apps/cleanup/management/commands/fix_xforms_with_undefined_xmlns.py]: E122 continuation line missing indentation or outdented

**Comment by benrudolph** [comment]
...this is the bug that never ends...yes, it goes on and on my friends....

File: corehq/apps/cleanup/management/commands/fix_xforms_with_undefined_xmlns.py [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+                   "{} were previously repaired. Build submitted against was {}"
+        super(MultiplePreviouslyFixedForms, self).__init__(template.format(app_id, build_id))
+
+
+@quickcache(["xform_instance.build_id"], memoize_timeout=ONE_HOUR)
+def get_correct_xmlns(xform_instance):
+    if xform_instance.build_id is None:
+        return None
+    build = get_app(xform_instance.domain, xform_instance.build_id)
+    # TODO: What if the app has been deleted?
```
Comment (czue) [inline corehq/apps/cleanup/management/commands/fix_xforms_with_undefined_xmlns.py]: then god help us

Comment (czue) [inline corehq/apps/cleanup/management/commands/fix_xforms_with_undefined_xmlns.py]: more seriously, i think it's probably fine to do nothing.

**Comment by czue** [comment]
nice, i was worried diving in but the way this was structured/broken up made it super easy to read through

**Pull request merged**